README.md

```diff
@@ -1,14 +1,91 @@
+
+---
+
+## 🌟 Key Features
+
+• Tri-lingual support (English, Sinhala, Tamil).
+• Expert verified medical information.
+• Free medical advice for managing rheumatic conditions.
+• Appointment booking system with rheumatology specialists.
+• Treatment options and medication guides.
+• User-friendly interface designed for accessibility.
+• Comprehensive resource library for various rheumatic conditions.
+• Mobile responsive design for access anywhere, anytime.
+
+---
+
+## 🚀 Current State
+RheumaConnect has successfully completed the design phase and the project is now in the development phase. Currently, the team is focused on building the platform's core features.
+
+### Roadmap
+Phase 1: Website development and initial content population
+Phase 2: Integration of multilingual support
+Phase 3: Continuous content updates and feedback integration
+
+---
+
+## 👥 Project Team
+
+### Project Initiators
+The project was initiated by the Software Engineering Student Community of SLIIT, in collaboration with medical professionals to bring this vision to life.
+
+### Core Team
+Software Engineering Student Community (SESC) - SLIIT
+Medical Specialists (Rheumatology Department)
+Volunteer Developers and Designers
+
+---
+
+## 🤝🏽 How You Can Contribute
+If you are passionate about making a difference and contributing to the project, we would love for you to get involved! You can contribute in several ways,
+
+### Technical Contributions
+
+• Frontend Development: Assist in designing the user interface for an intuitive and user-friendly experience.
+• Backend Development: Help in developing the server-side architecture and integration of multilingual capabilities.
+• UI/UX Design: Contribute to creating a visually appealing and accessible design for the platform.
+• Testing: Help with quality assurance by writing test cases, identifying and fixing bugs.
+
+### Non-Technical Contributions
+
+• Content Creation: Contribute by writing or curating informative articles for the platform, ensuring they are clear, concise, and    medically accurate.
+• Graphic Design: Help create engaging graphics, illustrations, and visual assets for the website and promotional materials.
+• Documentation: Assist in writing documentation for developers, users, and contributors to make the project easier to understand and participate in.
+• Community Outreach: Help spread the word about Rheumaconnect and engage with potential users and contributors.
+
+Please take a look at the **CONTRIBUTING.md** file if you would like to get a further understanding of how you can contribute.
+
+---
+
+### 🙏🏽 Acknowledgments
```

Comment:
This one let's reduce the content. Only keep the last line maybe, and add a line thanking everyone who takes part in the project, that would be sufficient.